RecoTauTag/Configuration/python/RecoTauTag_EventContent_cff.py

```diff
@@ -112,7 +113,9 @@
         'keep *_hpsPFTauMVA3IsolationChargedIsoPtSum_*_*',
         'keep *_hpsPFTauMVA3IsolationNeutralIsoPtSum_*_*',
         'keep *_hpsPFTauMVA3IsolationPUcorrPtSum_*_*',
-        'keep *_hpsPFTauTransverseImpactParameters_*_*'
+        'keep *_hpsPFTauTransverseImpactParameters_*_*',
+        'keep *_hpsPFTauNeutralIsoPtSumWeight_*_*',
+        'keep *_hpsPFTauPhotonPtSumOutsideSignalCone_*_*'
```

Comment:
it's about all modifications to this file RecoTauTag/Configuration/python/RecoTauTag_EventContent_cff.py
: they are not needed.